### ISSUE TITLE
New Content Suggestions

### ISSUE THREAD

**Issue by cavemansspa** [lhorie/mithril#43]
About content: a couple of endorsements from the last months in this channel: Fast, easy to learn for new developers, fun to work with, great documentation, Mithril is just javascript and helps you improve your understanding of JS.

"micro framework" is also a term to distinguish from angular and react

**Comment by cavemansspa** [comment]
WRT the mithril site, I don't think it's entirely a design issue. It is also a messaging problem.
When someone comes to the page, they want 1 question answered quickly.

"Why do I use mithril instead of a dozen other frameworks?" 

And I don't think that question is answered succinctly or clearly. Angular says ~"I'm just like HTML but better" - a great pitch for people who hate JS

Backbone said: ~"Traditional, Proven Structures brought to the browser" , which at the time was a great pitch when JS seemed like a fragile jquery Wild West.

Mithril says "Safe, Robust, Fast, Brilliant, Small" and also "A tool to organize your code"

So basically everything and nothing.

So what do we think is great about mithril, what sets it apart? Why are we all here? That has to be condensed into 3-4 words, and emphasised.

I think "A Javascript Framework for Building Brilliant Applications" doesn't actually say much.
The most on message aspect of mithril, is the name itself.
Mithril is a metal that is Light, Flexible, Strong. So emphasise that further. A lot of people won't know what Mithril is, so the connotation is lost.

**Comment by JAForbes** [comment]
If anyone and everyone involved in Mithril could just say in 1 word why they use mithril instead of another library.  Hopefully we could aggregate and come up with a clear message to help with a redesign of the site.  We are all here for a reason, the task is to make our reasons concise and precise.

**Comment by EirikBirkeland** [comment]
I agree that a good slogan is needed. Also, it might sound silly, but a memorable logo will help to solidify Mithril as a 'thing' in the community. This logo could be designed to both reinforce the core utility and purpose of Mithril as it stands, as well as play to the name. The Mithril framework is powerful but subtle, so I'd prefer something that is unimposing yet impressive.

**Comment by EirikBirkeland** [comment]
Some words: Minimalistic, functional, performant

**Comment by JAForbes** [comment]
Hey people, I've created an issue for this topic on the dev repo: lhorie/mithril.js#1026
